### PR TITLE
[PHP 8.1] Documentation for `full_path` entry of `$_FILES`

### PR DESCRIPTION
**Pull request by saundefined**
Add documentation for `full_path` entry, [introduced in PHP 8.1](https://www.php.net/manual/en/migration81.new-features.php#migration81.new-features.core.upload-full-path-key).

### CODE REVIEW

**Review by Alfreddo12 — APPROVED** [review]

**Review by Alfreddo12 — APPROVED** [review]

**Review by Alfreddo12 — APPROVED** [review]
> `
Duplicate of #`

**Comment by mumumu** [comment]
impl: https://github.com/php/php-src/pull/6917

File: features/file-upload.xml [hunk 1]
```diff
@@ -462,6 +471,37 @@ foreach ($_FILES["pictures"]["error"] as $key => $error) {
...
+]]>
+     </programlisting>
+    </example>
+
+    <warning>
+     <simpara>
+      PHP only parses the relative path information submitted by the browser/user-agent,
+      and passes that information to the <varname>$_FILES</varname> array.
+      There is no guarantee that the values in the <literal>full_path</literal> array contains a real directory structure,
+      and the PHP applications must not trust this information.
```
Comment (cmb69) [inline features/file-upload.xml:501]: nit: 
```suggestion
      and the PHP application must not trust this information.
```

**Review by cmb69 — COMMENTED** [review]
Thank you for the PR! This looks good to me, but maybe we want to add a warning that 
`webkitdirectory` is non-standard etc. like on https://developer.mozilla.org/en-US/docs/Web/API/HTMLInputElement/webkitdirectory.

File: features/file-upload.xml [hunk 2]
```diff
@@ -132,6 +132,15 @@
        </para>
       </listitem>
      </varlistentry>
+     <varlistentry>
+      <term><varname>$_FILES['userfile']['full_path']</varname></term>
+      <listitem>
+       <para>
+        The full path as submitted by the browser.
```
Comment (mumumu) [inline features/file-upload.xml:139]:
```suggestion
        The full path as submitted by the browser. This value does not always contain a real directory structure, and cannot be trusted.
```

We should also mention this value cannot be trusted here.

**Comment by saundefined** [comment]
Thanks! Added a warning about it

**Review by mumumu — APPROVED** [review]

**Pull request merged**